rclcpp/src/rclcpp/parameter_map.cpp

```diff
@@ -49,6 +51,17 @@ rclcpp::parameter_map_from(const rcl_params_t * const c_params)
       node_name = c_node_name;
     }
 
+    if (node_fqn) {
```

Comment:
okay, now i see. it would be more useful since this is done in `rclcpp::parameter_map_from`.